deploy/helm/values.schema.json

```
@@ -1312,6 +1319,9 @@
                 },
                 "resources": {
                     "$ref": "#/definitions/k8sResourceRequirements"
+                },
+                "storageClassName": {
+                    "$ref": "#/definitions/storageClassName"
```

related to the other comment:
```suggestion
                    "$ref": "#/definitions/k8sStorageClassName"
```